packages/mesos-client/src/observableFromXhrConnection.js

```diff
@@ -0,0 +1,30 @@
+import { ConnectionConstants, ConnectionManager } from "connection-manager";
+import Rx from "rxjs";
+
+export default function observableFromXhrConnection(connection) {
+  return Rx.Observable.create(function(observer) {
+    connection.addListener(
+      ConnectionConstants.CONNECTION_EVENT_PROGRESS,
```

Comment:
Do we need to prefix these constants with `CONNECTION_`? @orlandohohmeier @juliangieseke

Reply:
Adjusted in #2361